testsuite/smoke-test/src/operational_tooling.rs

```diff
@@ -1048,11 +1048,14 @@ async fn test_validator_set() {
 
 #[tokio::test]
 async fn test_verify_validator_state() {
-    let (_env, op_tool, backend, mut storage) = launch_swarm_with_op_tool_and_backend(4).await;
+    let (_env, op_tool, backend, mut storage) = launch_swarm_with_op_tool_and_backend(1).await;
 
     let result = op_tool.verify_validator_state(&backend).unwrap();
     assert_eq!(result.in_validator_set, Some(true));
     assert_eq!(result.consensus_key_match, Some(true));
+    assert_eq!(result.consensus_key_unique, Some(true));
```

Comment:
nit: you could assert that `is_state_consistent`/`is_valid_state` returns true here, right? Instead of all of these?

Reply:
Changed to `assert!(result.is_valid_state());`

This is much shorter! Thank you!